### PR TITLE
Add PHP file_exists

### PR DESCRIPTION
**Pull request by erikn69**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/locutusjs/locutus/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/locutusjs/locutus/pulls) for the same update/change?

### Description
Add PHP file_exists

### CODE REVIEW

**Comment by parallels999** [comment]
@kvz ping :)

**Comment by kvz** [comment]
Sorry for the late reply! I'm okay to merge this, but can you please run `yarn fix` in your branch?

**Comment by erikn69** [comment]
I don't remember where or why I needed this, but I imagine I solved it by another way.
I get `error locutus@2.0.29: The engine "yarn" is incompatible with this module.`

**Comment by kvz** [comment]
Ah sorry about that, I relaxed the constraints there this morning. I wonder if the lock file is gonna cause you issues you. If so, best use `corepack yarn` instead of `yarn` and I think on a recent Node.js, the correct yarn version will be used then.

**Comment by erikn69** [comment]
Done

File: src/php/filesystem/file_exists.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = function file_exists(filename) {
+  //       discuss at: https://locutus.io/php/file_exists/
+  //      original by: Erik Niebla
+  //           note 1: so this function is Node-only
+  //        example 1: var $bool = file_exists('test/never-change.txt')
```
Comment (kvz) [inline src/php/filesystem/file_exists.js:5]:
```suggestion
  //        example 1: file_exists('test/never-change.txt')
```

**Review by kvz — APPROVED** [review]

**Comment by kvz** [comment]
🎉  Thanks for your contribution!

**Pull request merged**